code/modules/cargo/packs/exploration.dm

```diff
@@ -0,0 +1,81 @@
+/datum/supply_pack/exploration
+	group = "Exploration"
+	crate_type = /obj/structure/closet/crate/wooden
+
+/*
+		Basic survival kits for worlds.
+*/
+
+/datum/supply_pack/exploration/lava
+	name = "Lava Exploration Kit"
+	desc = "Contains two pickaxes, 60 lavaproof rods, and goggles to protect eyes from the heat"
+	cost = 1500
+	contains = list(/obj/item/pickaxe/mini,
+					/obj/item/pickaxe/mini,
+					/obj/item/clothing/glasses/heat,
+					/obj/item/clothing/glasses/heat,
+					/obj/item/clothing/glasses/heat,
+					/obj/item/clothing/glasses/heat,
+					/obj/item/stack/rods/lava/thirty,
+					/obj/item/stack/rods/lava/thirty)
+	crate_name = "Lava Exploration Kit"
+
+/datum/supply_pack/exploration/ice
+	name = "Ice Exploration Kit"
+	desc = "Contains two pickaxes, winter clothes, and goggles to protect eyes from the cold"
+	cost = 1500
+	contains = list(/obj/item/pickaxe/mini,
+					/obj/item/pickaxe/mini,
+					/obj/item/clothing/glasses/cold,
+					/obj/item/clothing/glasses/cold,
+					/obj/item/clothing/glasses/cold,
+					/obj/item/clothing/glasses/cold,
+					/obj/item/clothing/shoes/winterboots,
+					/obj/item/clothing/shoes/winterboots,
+					/obj/item/clothing/shoes/winterboots,
+					/obj/item/clothing/shoes/winterboots)
+	crate_name = "Ice Exploration Kit"
+
+/datum/supply_pack/exploration/jungle
+	name = "Jungle Exploration Kit"
+	desc = "Contains hatchets, picks, and antivenom, great for dense jungles!"
+	cost = 750
+	contains = list(/obj/item/pickaxe/mini,
+					/obj/item/pickaxe/mini,
+					/obj/item/storage/pill_bottle/charcoal,
+					/obj/item/storage/pill_bottle/charcoal,
+					/obj/item/hatchet)
+	crate_name = "Jungle Exploration Kit"
+
+/datum/supply_pack/exploration/beach
+	name = "Beach Kit"
+	desc = "Shorts, picks, and (low quality) sunglasses, perfect for the beach!"
+	cost = 500
+	contains = list(/obj/item/pickaxe/mini,
+					/obj/item/pickaxe/mini,
+					/obj/item/clothing/under/shorts/black,
+					/obj/item/clothing/under/shorts/blue,
+					/obj/item/clothing/under/shorts/green,
+					/obj/item/clothing/under/shorts/grey,
+					/obj/item/clothing/under/shorts/purple,
+					/obj/item/clothing/under/shorts/red,
+					/obj/item/clothing/glasses/cheapsuns,
+					/obj/item/clothing/glasses/cheapsuns,
+					/obj/item/clothing/glasses/cheapsuns,
+					/obj/item/clothing/glasses/cheapsuns,
+					/obj/item/clothing/glasses/cheapsuns,
+					/obj/item/clothing/glasses/cheapsuns)
```

Comment:
generalize the formatting of those lists like
contains = list(
	obj1,
	obj2,
	obj3,
)